app.js

```diff
@@ -11,6 +12,7 @@ app.get('/api/categories', getCategories)
 app.get('/api/reviews/:review_id', getReview)
 app.get('/api/reviews', getReviews )
 app.get('/api/reviews/:review_id/comments', getComments)
+app.get('/api/users', getUsers)
 
 app.post('/api/reviews/:review_id/comments', postComment)
 
```

Comment:
Add an error handler for 404 - bad path. This isn't an error first/error handling middleware. Use the * wildcard.